library/Rules/Countable.php

```diff
@@ -13,8 +13,16 @@
 
 namespace Respect\Validation\Rules;
 
-class Countable extends AbstractRule
+/**
+ * @author Henrique Moody <henriquemoody@gmail.com>
```

Comment:
Can you add an line before telling what this rule does? You will find that in [Countable.md](https://github.com/Respect/Validation/blob/master/docs/Countable.md)